client/src/components/ManageArtist/ArtistForm.tsx

```diff
@@ -20,6 +22,9 @@ export interface ShareableTrackgroup {
   creatorId: number;
   slug: string;
 }
+const FormSectionWrapper = styled.div`
```

Comment:
And now this begets the question, could this be replaced with `<FormComponent>`? The argument for is that it would standardize the way form elements look! :)